.github/workflows/ci-test-debian.yml

```diff
@@ -21,6 +21,7 @@ jobs:
       CTEST_PARALLEL_LEVEL: 4
       IMAGE_TYPE: test
       BUILD_GENERATOR: Ninja
+      ENABLE_TOFINO: ON
```

Comment:
I would create a new job test-ubuntu22-tofino` which is a copy of this job and enable Tofino there. We can also disable all other back ends to save compilation time.